core/domain/stats_domain.py

```diff
@@ -1822,19 +1978,19 @@ def from_dict(cls, learner_answer_info_dict):
         )
 
     @classmethod
-    def get_new_learner_answer_info_id(cls):
+    def get_new_learner_answer_info_id(cls) -> str:
         """Generates the learner answer info domain object id.
 
         Returns:
             learner_answer_info_id: str. The id generated by the function.
         """
         learner_answer_info_id = (
             utils.base64_from_int(
-                utils.get_current_time_in_millisecs()) +
+                int(utils.get_current_time_in_millisecs())) +
```

Comment:
Why is this needed?

Reply:
Without casting as an int mypy reports the error `Argument 1 to "base64_from_int" has incompatible type "float"; expected "int"  [arg-type]`